cli/internal/cmd/trackingplan/validate/validate.go

```diff
@@ -30,52 +28,29 @@ func NewCmdTPValidate() *cobra.Command {
 			$ rudder-cli tp validate --loc <path-to-catalog-dir or file>
 		`),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println()
```

Comment:
I'm a lit bit skeptical of adding this Println before the output in selected commands only. If we're making this kind of presentation decisions, it's better if they are handled the same way across commands. Personally, I would prefer to skip it.